index.html

```diff
@@ -809,6 +809,21 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
 =&gt; [[1, 3, 5], [0, 2, 4]]
 </pre>
 
+      <p id="isEqual">
+        <b class="header">isEqual</b><code>_.isEqual(list, other)</code>
+        <br />
+        Performs an optimized deep comparison between one <b>list</b> and the <b>other</b>, to determine
```

Comment:
The documentation consistently uses **list** for every function in the *Collections* section. It serves as a shorthand for "array or object".